app/main/plugins.js

```diff
@@ -228,17 +230,15 @@ exports.queryResults = (plugin, args) => new Promise(resolve => {
     case 'dext':
       // no break
     default: { // eslint-disable-line no-fallthrough
-      const m = require(plugin.path); // eslint-disable-line global-require
-      let output = '';
-      if (typeof m.execute === 'function') {
-        output = m.execute(q);
-      } else {
-        output = m.execute;
-      }
-      let items = [];
+      // eslint-disable-next-line global-require
+      const pluginObj = require(plugin.path);
+      const output = (typeof pluginObj.execute === 'function')
+        ? pluginObj.execute(query, { size: MAX_RESULTS })
```

Comment:
Now `giphy` and other plugins can set a limit parameter on their respective apis 🎉